src/@types/users.ts

```diff
@@ -0,0 +1,23 @@
+export type Collaborator = SelectorItem<UserMini | GroupMini>;
+
+export type SelectorItem<T> = {
+    id: string;
+    item?: T; // ie UserMini or GroupMini
+    name: string;
+    value?: unknown;
```

Comment:
What is this used for?

Reply:
> What is this used for?

I don't know, just copied from BUIE. But I didn't find this field in the response. Maybe we can remove it.